### DISCUSSION TITLE
研究所推甄該選哪個英文能力考試？

### DISCUSSION THREAD

**Discussion by forestdew507** [1101-GS4719-C/Python-Community-GS4719-C#2225] [Q&A]
你選擇考哪個英檢的依據是什麼？

印象中全民英檢的有效期只有 2 年，而且只在台灣有效QQ，即使一些其他的選項 ( eg 劍橋 ) 沒有效期，但不少受理申請目標單位會要求要在 2 年內考取的檢定才算數。

想到接下來要準備申請研究所了，看起來對於有些所而言英文檢定並非必須，同時部分研究所也沒有針對英檢的相關簡章，好奇如果是拿來申請研究所證明自己的英文能力，大家認為必須嗎 ? 會優先選考哪一個英檢呢？( 似乎還是需要以自己擅長的為主齁哈哈XD 例如就有聽說有人認為多益偏向商用英文，如果對商用比較不熟悉就會相較難以取得優勢 )

**Comment by Jinyuu17** [comment]
為了學業的英文考試唯一推薦托福和雅思，基本上會承認其他的都會承認這種的考試。如果目標是申請美國或加拿大的大學，托福可能會更適合，特別是對學術英語的考察較為深刻。計劃移民到英國、澳大利亞等英聯邦國家，或者申請歐洲和亞洲的學校，雅思可能是更好的選擇，因為其在語言應用上的多樣性和認可度較高。

**Comment by Jinyuu17** [comment]
想申請台灣的研究所多益就很足夠了，聽讀測驗才1700元，相對托福雅思7700元的價格已經很划算了。考試內容也少了很多，準備起來比較簡單。

**Comment by yuan3x2** [comment]
台灣研究所的話多益應該就足夠了，但如果是想去外商公司實習或工作推薦托福或雅思成績，但相對的就是貴又難，最後如果是要出國讀書的話，托福和雅思就是必備了，去美國考托福，英國、澳洲等等可以考雅思。

**Comment by Chattenoire** [comment]
台灣選TOEIC，美國選TOEFL，大英帝國相關地區選IELTS，歐洲地區當然就是當地語言至少B1、理想是B2或以上。
至於全民英檢，除非你要找(高度需要英文的)公務員相關工作，那才要考，而且要考的是程度是高級。
